### PR TITLE
docs: update Cadence Pause blog post with electron 9 release

### PR DESCRIPTION
**Pull request by sofianguy**
To be in sync with Chromium, we have decided to move our Electron 9 stable date to Chromium's M83 stable date, May 19th.

ref https://github.com/electron/electron/pull/22863

### CODE REVIEW

File: data/blog/cadence-pause.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ We feel that our best choice is to follow in Chromium's footsteps, and so in the
 
 We want to ensure that both our maintainers and our consumers' wellbeing is prioritized during this time, so we welcome your feedback and look forward to returning to our regular release schedule.
 
-For more updates, please follow our [Twitter account](https://twitter.com/electronjs).
\ No newline at end of file
+For more updates, please follow our [Twitter account](https://twitter.com/electronjs).
+
+Edit: Electron 9 stable will target Chromium M83 and be released on May 19, 2020, in response to [Chromium's announcement](https://chromereleases.googleblog.com/2020/03/chrome-and-chrome-os-release-updates.html) of skipping M82 and M83 stable date.
```
Comment (malept) [inline data/blog/cadence-pause.md]: Based on my reading of the Chromium blog post:

```suggestion
Edit (2020-03-30): Electron 9 stable will target Chromium M83 and be released on May 19, 2020, in response to [Chromium's announcement](https://chromereleases.googleblog.com/2020/03/chrome-and-chrome-os-release-updates.html) of skipping the M82 stable date and postponing the M83 stable date.
```

**Comment by malept** [comment]
Should we also mention something along the lines of "Moving forward, we will be basing stable releases on odd-numbered Chromium releases" ?

**Comment by sofianguy** [comment]
I think we can omit that details to give us leeway in future release dates

**Comment by sofianguy** [comment]
@erickzhao good call. maybe we can add a line under that with something like "edit (2020-04-01): Electron 9 stable will be released on May 19th, 2020" ?

**Pull request merged**